scout/server/blueprints/institutes/templates/overview/cases.html

```diff
@@ -192,6 +192,54 @@
   </tr>
 {% endmacro %}
 
+{% macro sanger_to_be_validated(unevaluated, validated_by_others) %}
+ <div class="row container alert alert-info alert-dismissible" role="alert">
+  <div class="col-1">
+   <button type="button" class="close" data-bs-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+  </div>
+  <div class="col">
+    You have <strong>{{ unevaluated|length }}</strong> cases with Sanger validations to evaluate.
+    {% if validated_by_others == true %}
+      These variants have been validated by another user but they need also your validation since you ordered the Sanger sequencing.
+    {% endif %}
+    <!-- Button trigger modal -->
+    <button type="button" class="btn btn-primary btn-sm" data-bs-toggle="modal" data-bs-target="#evaluateModal_validated_{{validated_by_others}}">
+      evaluate
+    </button>
+  </div>
+
+  <!-- Modal -->
+  <div class="modal fade" id="evaluateModal_validated_{{validated_by_others}}" tabindex="-1" role="dialog" aria-labelledby="sangerModalLabel" aria-hidden="true">
```

Comment:
```suggestion
  <div class="modal fade" id="evaluateModal_validated{% if validated_by_others %}_by_others{% endif %}" tabindex="-1" role="dialog" aria-labelledby="sangerModalLabel" aria-hidden="true">
```
The construction here works, but is a bit unusual leaving the html a touch harder to read.